packages/mobile/src/transactions/transferFeedUtils.ts

```diff
@@ -106,7 +109,8 @@ export function getTransferFeedParams(
     recipientInfo
   )
   Object.assign(recipient, { address })
-  const nameOrNumber = recipient?.name || e164PhoneNumber
+  const nameOrNumber =
```

Comment:
Just a suggestion: extract this into a util function? it might be useful in other places as well?